src/v2/Apps/Settings/Routes/SavedSearchAlerts/Components/__tests__/SavedSearchAlertEditForm.jest.tsx

```diff
@@ -80,6 +84,9 @@ describe("SavedSearchAlertEditForm", () => {
         }
       }
     `,
+    variables: {
+      artistIDs: ["artist-id"],
+    },
   })
 
   it("renders pills", () => {
```

Comment:
good idea 👌

Reply:
Done. I had to replace `artists @relay(plural: true)` on `artistsConnection', since when using `@relay(plural: true)` it is not possible to return multiple array elements in tests

For example, when using this code
```javascript
renderWithRelay({
  Artists: () => [
    {
      name: "Artist #1"
    },
    {
      name: "Artist #2"
    },
  ],
})
```

In `artists` we will get automatically mocked data (the data we passed manually **was ignored**)
```javascript
artists [
  {
    internalID: '<mock-value-for-field-"internalID">',
    name: '<mock-value-for-field-"name">',
    slug: '<mock-value-for-field-"slug">'
  }
]
```

Reply:
Also please pay attention to [this comment](https://github.com/artsy/force/pull/10301#discussion_r897844501)

Reply:
huh interesting! good to know 🤔 thanks for making that update!